core/src/main/java/org/apache/carbondata/core/view/MVProvider.java

```diff
@@ -151,40 +129,34 @@ public MVSchema getSchema(MVManager viewManager,
   }
 
   void saveSchema(MVManager viewManager, String databaseName,
-                  MVSchema viewSchema) throws IOException {
```

Comment:
please revert method signature changes in all places of this class

Reply:
OK